### PR TITLE
Update readme

### LINKED ISSUE #588: Improve Readme
## Feature Request
Improve Readme 

#### Feature Description
Improve Readme file with play store account link, app logo, chat badge and Used libraries description which can help newcomers to get easy access and it makes readme file look appropriate

### PR DESCRIPTION
**Pull request by anandwana001**
Fixes #588 

Changes: Improve Readme file with play store account link, app logo, chat badge and Used libraries description

### CODE REVIEW

**Comment by anandwana001** [comment]
Could you help me find where it is written!

**Comment by srv-twry** [comment]
It should be in the codestyle or commit guidelines. Codestyle most probably. Link to the CODESTYLE is available in the README or you can check the docs folder.

**Comment by anandwana001** [comment]
@srv-twry Required changes done

File: docs/codestyle.md [hunk 1]
```diff
@@ -30,7 +30,6 @@ Try to remove as much warnings as possible, It's not completely possible to remo
 
 ### Some **DONT's**
 
-- Don't use Hungarian Notation like `mContext` `mCount` etc
```
Comment (mhutti1) [inline docs/codestyle.md]: This is still valid so lets keep it.

**Comment by anandwana001** [comment]
@mhutti1 Ok
@srv-twry can we merge this pull request by keeping the previous commit [Improve Readme](https://github.com/kiwix/kiwix-android/pull/589/commits/32afdf19c81dc50f6767df8c4fc21d953f2f53bc) and removing [Last commit](https://github.com/kiwix/kiwix-android/pull/589/commits/e8f73f060885a2586968e7020c76eb9fa2024c30)
is it possible? How?

**Pull request merged**